student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
```

Comment:
- [ ] Your initialize should look something like below to give your student a name and age and parent permisison right :+1: apart from just default values right :thinking:

```
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Reply:
@HAMMAS-SALEEM  thanks

fixed